codex-cli/src/approvals.ts

```diff
@@ -333,7 +333,9 @@ export type SafeCommandReason = {
 export function isSafeCommand(
   command: ReadonlyArray<string>,
 ): SafeCommandReason | null {
-  const [cmd0, cmd1, cmd2, cmd3] = command;
+  // Trim out any whitespace‑only tokens so leading spaces don’t break matching
```

Comment:
This is not a string of shell script: it is an array of arguments. Whitespace is significant in each argument. If it's there, it must be honored.

What did you observe in the wild that prompted this change?